### PR TITLE
Call out cert-manager not working on AWS Fargate

### PR DESCRIPTION
**Pull request by GeoffWilliams**
Fargate is not possible due to https://github.com/jetstack/cert-manager/issues/3237 - PR to docs as requested in ticket

### CODE REVIEW

File: content/en/docs/installation/compatibility.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ deployment, or, if using Helm, configuring it in your `values.yaml` file.
 Note that since kubelet uses port `10250` by default on the host network, the
 `webhook.securePort` value must be changed to a different, free port.
 
+Cert-manager is not compatible with AWS Fargate, it must be run inside a
```
Comment (meyskens) [inline content/en/docs/installation/compatibility.md:46]:
```suggestion
cert-manager is not compatible with AWS Fargate, it must be run inside a 
```

File: content/en/docs/installation/compatibility.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ deployment, or, if using Helm, configuring it in your `values.yaml` file.
 Note that since kubelet uses port `10250` by default on the host network, the
 `webhook.securePort` value must be changed to a different, free port.
 
+Cert-manager is not compatible with AWS Fargate, it must be run inside a 
+nodegroup. For more information see
```
Comment (meyskens) [inline content/en/docs/installation/compatibility.md:47]: must be run feels weird to me... could just be me :)

Comment (GeoffWilliams) [reply content/en/docs/installation/compatibility.md:47]: open to corrections :)

Comment (meyskens) [reply content/en/docs/installation/compatibility.md:47]: How about

```
cert-manager is not compatible with AWS Fargate, in order to use it it must be placed in a nodegroup
```

Hope that sounds better? However never having tried fargate i have no idea if this is correct :)

Comment (GeoffWilliams) [reply content/en/docs/installation/compatibility.md]: accepted.
* re-based against `master`
* squashed commits
* added `Signed-off-by`

Fargate is pretty sick, IMO its the future of Kubernetes on AWS. Its a fully managed service with no EC2 VMs exposed to users so there's less things to worry about or tinker with. The downside of this is that Kubernetes `Nodeport` and `DaemonSet` simply don't exist on this platform, so systems must be implemented to take account of this. :sunglasses:

Comment (meyskens) [reply content/en/docs/installation/compatibility.md]: I never liked EKS personally it was too much work t be on a cloud... Thanks for the help!

**Comment by SgtCoDFish** [comment]
Hey @GeoffWilliams - it looks like this PR got a little lost in the backlog. :sweat_smile: 

Fortunately it also looks like cert-manager can actually run on AWS Fargate according to [this post](https://github.com/jetstack/cert-manager/issues/3237#issuecomment-827523656). Looks like an unfortunate port clash which ended up presenting in a confusing way.

I'm going to close this PR, since I'll be opening a new one to add docs reflecting what's said in [the post](https://github.com/jetstack/cert-manager/issues/3237#issuecomment-827523656).

Thanks again for raising this PR, though :+1:

**Pull request closed without merging**